src/Fixer/ReturnNotation/ReturnAssignmentFixer.php

```diff
@@ -278,6 +278,20 @@ private function fixFunction(Tokens $tokens, int $functionIndex, int $functionOp
                 continue;
             }
 
+            // Check if there is a finally block and if the variable is used in it
+            $tryIndex = $tokens->getPrevTokenOfKind($returnVarIndex, [[T_TRY]]);
+            if (null !== $tryIndex && $tryIndex > $functionOpenIndex && $tryIndex < $functionCloseIndex) {
```

Comment:
Can we have this extracted to a separate method to have it easier to comprehend in the future?